roles/confluent.schema_registry/tasks/main.yml

```diff
@@ -105,6 +105,13 @@
     group: "{{schema_registry_group}}"
   notify: restart schema-registry
 
+- name: Enable Secrets Protection
```

Comment:
Because this task only runs once, ie can only encrypt a single file per component. I think we should just assume it's going to encrypt the component's properties file. And therefore we should either default this variable
```
schema_registry_secrets_protection_config_path: "{{schema_registry.config_path}}"
```

Or we could force this task to only work w the properties file which would look like 
```
- name: Enable Secrets Protection
  include_tasks: ../../tasks/secrets_protection.yml
  vars:
    params: "{{ schema_registry_secrets_protection_config_params }}"
    config_path: "{{ schema_registry.config_path }}"
  when: schema_registry_secrets_protection_enabled|bool
```